apps/frontend/components/field/referee/inspection-timer.tsx

```diff
@@ -0,0 +1,64 @@
+import { useMemo } from 'react';
+import dayjs from 'dayjs';
+import { Paper, Typography, Box } from '@mui/material';
+import Countdown from '../../general/countdown';
+import { useTime } from '../../../hooks/use-time';
+
+interface InspectionTimerProps {
+  startTime: string;
+}
+
+const INSPECTION_LENGTH = 90; // 1.5 minutes in seconds
+
+const InspectionTimer: React.FC<InspectionTimerProps> = ({ startTime }) => {
+  const currentTime = useTime({ interval: 100 });
+  const inspectionEnd = dayjs(startTime).add(INSPECTION_LENGTH, 'seconds');
+
+  const percentLeft = useMemo(
+    () => inspectionEnd.diff(currentTime) / (10 * INSPECTION_LENGTH),
+    [currentTime, inspectionEnd]
+  );
+
+  if (percentLeft <= 0) {
+    return null;
+  }
+
+  return (
+    <Box
+      sx={{
+        position: 'fixed',
+        bottom: 16,
+        right: 'auto',
+        left: 16,
+        zIndex: 1000,
+      }}
+    >
+      <Paper
+        elevation={6}
+        sx={{
+          py: 1,
+          px: 2,
+          bgcolor: 'info.main',
```

Comment:
White background please, you can add a colorful shadow like in `welcome-header.tsx`